frontend/src/components/applications/StaffingCertificateInput.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <v-form ref="form">
+    <v-row no-gutters class="mt-8">
+      <v-col v-for="(certificate, index) in certificates" :key="index" cols="12" sm="6" lg="4" xl="3" class="px-6 px-lg-8 mb-2" align="center">
+        <v-row no-gutters>
+          <v-col cols="12" sm="1" class="pt-2">{{ index + 1 }}.</v-col>
+          <v-col cols="12" sm="4" class="pt-1">
+            <v-text-field
+              v-model.trim="certificate.initials"
+              variant="outlined"
+              density="compact"
+              :disabled="readonly"
+              placeholder="Initials"
+              maxlength="4"
+              rounded="0"
+              :rules="rules.required"
+              :error-messages="getErrorMessagesForDuplicateError(certificate.isInitialsDuplicate)"
+              @input="update(certificate)"
+              @blur="checkDuplicateInitials" />
+          </v-col>
+          <v-col cols="12" sm="6" class="pt-1">
+            <v-text-field
+              v-model="certificate.certificateNumber"
+              variant="outlined"
+              density="compact"
+              :disabled="readonly"
+              :placeholder="certificateNumberPlaceholder"
+              maxlength="6"
+              rounded="0"
+              :rules="rules.required"
+              :error-messages="getErrorMessagesForDuplicateError(certificate.isCertificateNumberDuplicate)"
+              @input="update(certificate)"
+              @blur="checkDuplicateCertificateNumbers" />
+          </v-col>
+          <v-col v-if="certificate.initials || certificate.certificateNumber" cols="12" sm="1" class="pl-3" align="center">
+            <v-btn icon="mdi-trash-can-outline" variant="text" @click="remove(index)" />
```

Comment:
The v-btn has a weird squished oval overlay on hover:
![image](https://github.com/bcgov/ecc-ofm/assets/80127167/d434ae6a-e390-4a24-8023-e01f3755c475). Maybe try with child icon which looks on on ApplicationsHistoryView.vue.

Reply:
it's updated